content/blaze.md

```diff
@@ -0,0 +1,543 @@
+      style: {background: 'blue'}
+    };
+  }
+});
+```
+
+<h3 id="rendering-html">Rendering pure HTML</h3>
+Although by default a mustance tag won't render any HTML, for XSS security reasons, you can render pure HTML with the triple-mustache: `{{{`.
+
+```html
+{{{myHtml}}}
+```
+
+```js
+Template.foo.helpers({
+  myHtml() {
+    return '<h1>This H1 will render</h1>';
+  }
+});
+```
+
+You should be extremely careful about doing this, and always ensure you aren't returning user-generated content (or escape it if you do!) from such a helper.
+
+<h3 id="block-helpers">Block Helpers</h3>
+A block helper, called with `{{#` is a helper that takes (and may render) a block of Spacebars. For instance, we saw the `{{#each in}}` helper above which repeats a given block of Spacebars once per item in a list. You can also render a *template* as a block helper, rendering it's content via the `Template.contentBlock` and `Template.eachBlock`. For instance, you could create your own `{{#if}}` helper with:
+
+```html
+<template name="myIf">
+  {{#if condition}}
+    {{> Template.contentBlock}}
+  {{else}}
+    {{> Template.elseBlock}}
+  {{/if}}
+</template>
+
+<template name="caller">
+  {{#myIf condition=true}}
+    <h1>I'll be rendered!</h1>
+  {{/myIf}}
+```
+
+<h3 id="builtin-block-helpers">Builtin Block Helpers</h3>
+There are a few builtin block helpers that are worth knowing about:
+
+<h4 id="if-unless">If / Unless</h4>
+
+The `{{#if}}` and `{{#unless}}` helpers are fairly straightforward but invaluable for controlling the content generated by a template. Both operate by evaluating and checking their single argument for "falsey"-ness (in JS this means `null`, `undefined`, `0`, `''`, `[]` and of course `false`).
+
+```html
+{{#if something}}
+  <p>It's true</p>
+{{else}}
+  <p>It's false</p>
+{{/if}}
+```
+
+<h4 id="each-in">Each-in</h4>
+
+The `{{#each in}}` helper is a convenient way to step over a list whilst retaining the outer data context. 
+
+```html
+{{#each todo in todos}}
+{{/each}}
+```
+
+In this case `todo` will be added to the data context within the block, but all the existing data context items (`list` and `todosReady` in this case) will remain available.
+
+<h4 id="let">Let</h4>
+The `{{#let}}` helper is useful to capture the output of a helper or document subproperty within a template:
+
+```html
+{{#let name=person.bio.firstName color=generateColor}}
+  <div>{{name}} gets a {{color}} card!</div>
+{{/let}}
+```
+
+XXX: what to do about d.c. changing helpers such as with/each?
+
+<h4 id="strictness">Strictness</h4>
+Spacebars starts from a very strict interpretation of HTML. In particular you need to careful about self-closing tags and when you are allowed to use them. For instance, you can't self-close a `div` (`<div/>`) in Spacebars.
+
+<h4 id="escaping">Escaping</h4>
+To insert a literal {{, {{{, or any number of curly braces, put a vertical bar after it. So `{{|` will show up as `{{`, `{{{|` will show up as `{{{`, and so on.
```

Comment:
Whoa, I had no idea... this would really simplify some stuff in the docs app lol